api_tests/licenses/views/test_license_detail.py

```diff
@@ -1,29 +1,28 @@
-from nose.tools import *  # flake8: noqa
+import pytest
 import functools
 
-from tests.base import ApiTestCase
-from osf.models.licenses import NodeLicense
 from api.base.settings.defaults import API_BASE
+from osf.models.licenses import NodeLicense
 
+@pytest.mark.django_db
+class TestLicenseDetail:
 
-class TestLicenseDetail(ApiTestCase):
-    def setUp(self):
-        super(TestLicenseDetail, self).setUp()
-        self.license = NodeLicense.find()[0]
-        self.url = '/{}licenses/{}/'.format(API_BASE, self.license._id)
-        self.res = self.app.get(self.url)
-        self.data = self.res.json['data']
+    def test_license_detail(self, app):
+        license = NodeLicense.find()[0]
```

Comment:
Minor; license is a constant from the site module (thus why it is highlighted funny; see [this link if you wanna know more](https://docs.python.org/2/library/constants.html#constants-added-by-the-site-module)). Probably rename, though it is unlikely to break anything.

Also, see note about .find() below.